suplemon/ui.py

```diff
@@ -326,13 +327,17 @@ def show_top_status(self):
             head_parts.append(self.file_list_str())
 
         head = " ".join(head_parts)
-        head = head + (" " * (self.screen.getmaxyx()[1]-len(head)-1))
-        if len(head) >= size[0]:
-            head = head[:size[0]-1]
-        if self.app.config["display"]["invert_status_bars"]:
-            self.header_win.addstr(0, 0, head, curses.color_pair(0) | curses.A_REVERSE)
-        else:
-            self.header_win.addstr(0, 0, head, curses.color_pair(0))
+        head = head + (" " * (size[0]-wcswidth(head)-1))
+        head_width = wcswidth(head)
+        if head_width > size[0]:
+            head = head[:size[0]-head_width]
+        try:
+            if self.app.config["display"]["invert_status_bars"]:
+                self.header_win.addstr(0, 0, head, curses.color_pair(0) | curses.A_REVERSE)
+            else:
+                self.header_win.addstr(0, 0, head, curses.color_pair(0))
+        except:
```

Comment:
As mentioned in another issue, it would be good to be explicit about what this `except` is catching. For example, it could accidentally catch a human error (e.g. missing `"display"` key in `self.app.config`)

Reply:
Thanks for pointing this out. Fixed.